content/polyhedra/content.md

```diff
@@ -64,20 +64,20 @@ We already know that the sum of the internal angles in a triangle is always
       circle.move(name="b" cx=260 cy=80)
       circle.move(name="c" cx=240 cy=220)
       circle.move(name="d" cx=80 cy=260)
-      path.fill.red(x="angle(b,a,d)" label="${round(angle(b,a,d).deg)}°")
-      path.fill.blue(x="angle(c,b,a)" label="${round(angle(c,b,a).deg)}°")
-      path.fill.green(x="angle(d,c,b)" label="${round(angle(d,c,b).deg)}°")
-      path.fill.yellow(x="angle(a,d,c)" label="${round(angle(a,d,c).deg)}°")
+      path.fill.red(x="geometryLeft.a.exact" label="${geometryLeft.a.rounded}°")
+      path.fill.blue(x="geometryLeft.b.exact" label="${geometryLeft.b.rounded}°")
+      path.fill.green(x="geometryLeft.c.exact" label="${geometryLeft.c.rounded}°")
+      path.fill.yellow(x="geometryLeft.d.exact" label="${geometryLeft.d.rounded}°")
       path(x="segment(a,b)")
       path(x="segment(b,c)")
       path(x="segment(c,d)")
       path(x="segment(d,a)")
```

Comment:
Rather than drawing each segment individually, you could draw the entire polygon in one. Then you can also give it a name, to access in the TypeScript code, rather than passing around arrays of points. For example,

```
path(x="polygon(a,b,c,d)" name="leftPolygon")
```

Then, we could add a `.internalAngles` property to the `Polygon` class in fermat.js, which returns an
array of angles. I think that would make the code below much more readable.